documentation/api/query/v4/facts.markdown

```diff
@@ -80,7 +80,7 @@ Subquery against `/fact-contents` to get all remotely-authenticated trusted fact
 
     curl -X GET http://localhost:8080/v4/facts --data-urlencode 'query=["in", ["name","certname"],
       ["extract",["name","certname"],
-        ["select-fact-contents", ["~>", "path", [".*", "authenticated"]]]]]'
+        ["select_fact_contents", ["~>", "path", [".*", "authenticated"]]]]]'
```

Comment:
Looking at this, it obviously points out we need to redesign this operator a bit better so the keyword, and the thing its operated on is separate. More importantly, this points out something to me ... if we don't change our endpoints to use underscores, this now doesn't map correctly, as it uses underscores whereas the endpoint uses hyphens.